flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/factories/TableFactoryUtil.java

```diff
@@ -47,15 +48,14 @@ public static ExternalCatalog findAndCreateExternalCatalog(Descriptor descriptor
 	 */
 	public static <T> TableSource<T> findAndCreateTableSource(Descriptor descriptor) {
 		Map<String, String> properties = descriptor.toProperties();
-
```

Comment:
nit: revert this unnecessary change?

Reply:
I guess it's okay. It's more consistent with findAndCreateTableSink().